modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -438,6 +440,9 @@ PropertyInfo VisualScriptBuiltinFunc::get_input_value_port_info(int p_idx) const
 		case TYPE_EXISTS: {
 			return PropertyInfo(Variant::STRING, "type");
 		} break;
+		case TEXT_ORD: {
```

Comment:
It would be better if this case just after `case TEXT_CHAR` to be consistent with line 105.

Otherwise the changes look good. The commits should be squashed together before merge though (see [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase)).